packages/app-desktop/gui/getWindowTitle.ts

```diff
@@ -0,0 +1,23 @@
+const Setting = require('@joplin/lib/models/Setting').default;
+import BaseModel from '@joplin/lib/BaseModel';
+import { _ } from '@joplin/lib/locale';
+const { substrWithEllipsis } = require('@joplin/lib/string-utils');
+
+function getWindowTitle(props: any) {
```

Comment:
We need to define what arguments this function takes, so please either create an interface for `props` that tells what props are actually used by the function, or create separate arguments for each dependency.